java/jtest/README.md

```diff
@@ -0,0 +1,127 @@
+# HDF5 Java FFM Tests
+
+FFM (Foreign Function & Memory) API tests for HDF5 Java bindings.
+
+## Test Files
+
+Tests use direct FFM bindings to HDF5 C APIs:
+- `TestH5*ffm.java` - FFM tests for each H5 module
+- `FfmTestSupport.java` - Shared FFM utilities and patterns
+
+## Current Coverage (435 tests - 16/18 modules)
```

Comment:
This table will just become unmaintained

Reply:
Yes will eliminate